src/components/message/MessagesList.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Image, FlatList } from 'react-native';
+import PropTypes from 'prop-types';
+
+import MessageRow from '../message/MessageRow';
+
+
+const imageStyle = {
+    height: 40,
+    width: 40,
+    borderRadius: 20,
+    marginRight: 10
+};
```

Comment:
can we add TODO's here to remind us to make these pixel sizes dynamic (aka based on screen size and not fixed)

Reply:
https://facebook.github.io/react-native/docs/dimensions.html